api/models/templates/app.tmpl

```diff
@@ -287,24 +249,28 @@
         "Default" : "{{ index $balancer.Randoms .Balancer }}",
         "Description" : ""
       },
+      {{ if .Secure }}
```

Comment:
I think these conditionals will be problematic if the parameter is not there at all. We should either fully de-parameterize or make them unconditional I think.